packages/chart/src/Axis.ts

```diff
@@ -587,53 +587,50 @@ export class Axis extends SVGWidget {
             .attr("class", "axisTitle")
             .merge(svgText)
             ;
+        const svgTitleTransition = svgTextUpdate.transition()
+            .attr("dx", null)
+            .style("text-anchor", "end")
+            ;
         switch (this.orientation()) {
             case "left":
-                svgTextUpdate.transition()
+                svgTitleTransition
                     .attr("transform", "rotate(-90)")
                     .attr("x", -2)
                     .attr("y", 2)
-                    .attr("dx", null)
                     .attr("dy", ".71em")
-                    .style("text-anchor", "end")
-                    .text(this.title())
                     ;
                 break;
             case "right":
-                svgTextUpdate.transition()
+                svgTitleTransition
                     .attr("transform", "rotate(-90)")
                     .attr("x", -2)
                     .attr("y", 4)
                     .attr("dx", null)
                     .attr("dy", "-.71em")
-                    .style("text-anchor", "end")
-                    .text(this.title())
                     ;
                 break;
             case "top":
-                svgTextUpdate.transition()
+                svgTitleTransition
                     .attr("transform", "rotate(0)")
                     .attr("x", svgLineBBox.width - 2)
                     .attr("y", 2)
                     .attr("dx", null)
                     .attr("dy", ".71em")
-                    .style("text-anchor", "end")
-                    .text(this.title())
                     ;
                 break;
             case "bottom":
-                svgTextUpdate.transition()
+                svgTitleTransition
                     .attr("transform", "rotate(0)")
                     .attr("x", svgLineBBox.width - 2)
                     .attr("y", -2)
-                    .attr("dx", null)
                     .attr("dy", null)
-                    .style("text-anchor", "end")
-                    .text(this.title())
                     ;
                 break;
             default:
         }
+        svgTitleTransition
+            .text(this.title_exists() ? this.title() : "")
```

Comment:
`this.title()` is fine here as it will either be "" or what the user has set.
(the same is not true in XYAxis)